src/status_im/chat/models/reactions.cljs

```diff
@@ -105,3 +107,42 @@
        acc))
    []
    reactions))
+
+(defn- <-rpc
+  [emoji-reaction]
+  (-> emoji-reaction
+      (set/rename-keys
+       {:emojiId       :emoji-id
+        :compressedKey :compressed-key})
+      (select-keys [:emoji-id :from :compressed-key])))
+
+(defn- format-response
+  [response]
+  (->> (transforms/js->clj response)
+       (map <-rpc)
+       (group-by :emoji-id)
+       (into (sorted-map))))
+
+(rf/defn save-emoji-reaction-details
+  {:events [:chat/save-emoji-reaction-details]}
+  [{:keys [db]} message-reactions long-pressed-emoji]
+  {:db (-> db
+           (assoc-in [:chats (:current-chat-id db) :message/reaction-authors-list] message-reactions)
+           (assoc-in [:chats (:current-chat-id db) :message/reaction-authors-list-selected-reaction]
+                     long-pressed-emoji))})
+
+(rf/defn clear-emoji-reaction-details
+  {:events [:chat/clear-emoji-reaction-author-details]}
+  [{:keys [db]} message-reactions]
+  {:db (update-in db [:chats (:current-chat-id db)] dissoc :message/reaction-authors-list)})
```

Comment:
i see that sometimes chat-id is used and sometimes current-chat-id we need a consistency here